landoapi/notifications.py

```diff
@@ -48,3 +51,43 @@ def notify_user_of_landing_failure(email, revision, error, request_id):
         # from the Transplant service then the HTTP error code will cause the Transplant
         # service to wait and retry the request.
         raise
+
+
+def notify_user_of_bug_update_failure(email, revision, error, request_id):
+    """Send out user notifications that a bug update failed.
+
+    Args:
+        email (str): Receipent's email address (e.g. `requester_email`.)
+        revision (str): The revision associated with the bug update failure.
+        error (str): The text of the error associated with the bug update failure.
+        request_id (int): A unique identifier identifying either a legacy request ID or
+            a LandingJob ID.
```

Comment:
```suggestion
        request_id (int): A unique identifier identifying a LandingJob ID.
```

Legacy requests are fully deprecated, and just need to be ripped out of the code base.